src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  contructor(name) {
```

Comment:
Typo here should be
```suggestion
  constructor(name) {
```

Reply:
Ohhh that's why it wasn't working as i tought it should work with inheritance. Now it all makes sense :D